### PR TITLE
Add Lohrenz-Bray-Clark viscosity correlation

### PR DESCRIPTION
**Pull request by joshua-white**
This PR adds an LBC viscosity correlation to our compositional fluid property calculations.

### CODE REVIEW

File: src/coreComponents/schema/docs/CompositionalMultiphaseFluidLohrenzBrayClarkViscosity.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+
+============================ ============== ======== ============================================== 
+Name                         Type           Default  Description
```
Comment (herve-gross) [inline src/coreComponents/schema/docs/CompositionalMultiphaseFluidLohrenzBrayClarkViscosity.rst:4]: In the description, we should have units where appropriate.

**Review by herve-gross — APPROVED** [review]
Great! This is a good way to expand our PVT toolbox. Minor comment regarding units.

**Comment by rrsettgast** [comment]
@joshua-white What are we doing with this PR?

**Comment by joshua-white** [comment]
It is still active.  I need to add two features, and could probably have it ready to merge next week:
- Add a ConstantViscosity model so we can continue to pass the current baselines
- Expose a mixing rule choice in the XML (Herning-Zipperer vs. Wilke vs. Brokaw)

**Comment by dkachuma** [comment]
Closing as this was merged in #2879

**Pull request closed without merging**